src/sql/mod.rs

```diff
@@ -167,6 +167,9 @@ mod tests {
         WITH t1 AS (SELECT a,d FROM table_1),
         t2 AS (SELECT * FROM table_2)
         SELECT * FROM t1 INNER JOIN t2 ON t1.d = t2.x INNER JOIN table_2 ON t1.d=table_2.x ORDER BY t1.a, t2.x, t2.y, t2.z LIMIT 17",
+        "SELECT CASE a WHEN 5 THEN 0 ELSE a END FROM table_1",
+        "SELECT CASE WHEN a < 5 THEN 0 WHEN a < 3 THEN 3 ELSE a END FROM table_1",
+        //"SELECT CASE WHEN a < 5 THEN 0 WHEN a < 3 THEN 3 END FROM table_1",
```

Comment:
Why do we have this commented?

Reply:
Because for the last one we need to implement Value::Null first